ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/util/SynskarpaEnum.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.ts_bas.v7.model.converter.util;
+
+public enum SynskarpaEnum {
```

Comment:
Maybe this enum should be defined together with the QuestionSynskarpa. It is only used in order to specify what value to retrieve when calling toInternal.

Reply:
Refactored into a inner enum in QuestionSynskarpa and renamed to VisualAcuityEnum